### PR TITLE
Clean out usage of posix header files

### PR DESCRIPTION
**Pull request by JCash**
Code cleanup task

### CODE REVIEW

File: engine/dlib/src/zip/miniz.h [hunk 1]
```diff
@@ -1,17 +1,3 @@
...
-// this file except in compliance with the License.
-// 
-// You may obtain a copy of the License, together with FAQs at
-// https://www.defold.com/license
-// 
-// Unless required by applicable law or agreed to in writing, software distributed
-// under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
-// CONDITIONS OF ANY KIND, either express or implied. See the License for the
-// specific language governing permissions and limitations under the License.
-
```
Comment (AGulev) [inline engine/dlib/src/zip/miniz.h:14]: Could you please add these files into [apply_license.py](https://github.com/defold/defold/pull/7412/files#diff-7517c4d65e6a229d0fc054796d485f61554e8c0a790d440ff865457e9546010e) to ignore it in the future?

Comment (JCash) [reply engine/dlib/src/zip/miniz.h:14]: Thx, I was looking to find that one...

File: engine/engine/src/engine.cpp [hunk 2]
```diff
@@ -17,6 +17,7 @@
 #include "engine_private.h"
 
 #include <dmsdk/dlib/vmath.h>
+
 #include <sys/stat.h>
```
Comment (Jhonnyg) [inline engine/engine/src/engine.cpp:21]: Should this still be here?

**Review by Jhonnyg — APPROVED** [review]
Looks good! Just a small q

File: engine/engine/src/engine.cpp (hunk 2 above)
Comment (JCash) [reply engine/engine/src/engine.cpp:21]: I have other changes in the private repo as well for this case. So it's a two-step process.

**Pull request merged**